client/verta/docs/tutorials/packaging/create_registered_model.rst

```diff
@@ -8,15 +8,15 @@ This tutorial will go through how to create a registered model using the Client
 Using the client
 ----------------
 
-Registered models can be created as follows:
+Registered models can be created with :meth:`Client.create_registered_model() <verta.client.Client.create_registered_model()>` as follows:
```

Comment:
This probably isn't a syntax error, but for consistency:
```suggestion
Registered models can be created with :meth:`Client.create_registered_model() <verta.client.Client.create_registered_model>` as follows:
```

Reply:
Fixed.